assembly/package.json

```diff
@@ -70,7 +70,7 @@
   "scripts": {
     "prepare": "yarn run clean && yarn build",
     "clean": "theia clean && rimraf errorShots",
-    "build": "theia build --mode development --config cdn/webpack.config.js --env.cdn=./cdn.json --env.monacopkg=@theia/monaco-editor-core@0.20.0 && yarn run override-vs-loader",
+    "build": "NODE_OPTIONS='--max-old-space-size=4096' theia build --mode development --config cdn/webpack.config.js --env.cdn=./cdn.json --env.monacopkg=@theia/monaco-editor-core@0.20.0 && yarn run override-vs-loader",
```

Comment:
I would move it to `.github/workflows/pr.yml` ENV variable

Reply:
I've thought about this a bit more, and I think it's correct to set this only for the theia build: the size of memory needed depends on the task being executed, not the environment. Other tasks don't need that much max memory, and giving it to them can make them grab more than they need and take the memory away from other processes, thus reducing potential concurrency.